src/explore-education-statistics-common/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -36,7 +42,14 @@ interface Props {
     indicators: string[];
     filters: string[];
   };
+  selectedPublication?: SelectedPublication;
+  subject: Subject;
   subjectMeta: SubjectMeta;
+  tableSizeErrorDownloadAvailable?: boolean;
```

Comment:
This is a bit of a mouthful with the 'available' - could simplify this to `showTableSizeErrorDownload`

Reply:
Happy that this has been addressed :)